src/test/java/com/radixdlt/identifiers/EUIDTest.java

```diff
@@ -95,7 +95,6 @@ public void testGetLow() {
 	public void testGetShard() {
 		EUID euid = new EUID(UInt128.from(2L, 5L));
 		assertEquals(5L, euid.getLow());
-		assertEquals(2L, euid.getShard());
```

Comment:
This test is really a no-op now, and can be removed.  `getLow()` is tested in `testGetLow()`.

Reply:
Done